doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
+
+It is also possible to build a Flatpak package containing Elektra:
+
+```bash
+# install flatpak and flatpak-builder, using the package manager of your distro
+$ apt install flatpak flatpak-builder
+
+# download the runtime and sdk
+$ flatpak install flathub org.kde.Platform//5.15 org.kde.Sdk//5.15
+
+# build and install the package for the current user
+$ flatpak-builder .flatpak-app scripts/flatpak/org.libelektra.kdb.yaml  --force-clean --user --install
+
+# try running the kdb binary
+$ flatpak run org.libelektra.kdb
+
+# add an alias to your shell
+$ echo 'alias kdb="flatpak run org.libelektra.kdb"' >> ~/.bashrc && source ~/.bashrc
+```
```

Comment:
man (help) pages do not seem to work:

```
markus@markusbyte:~floss/aufgaben> XDG_DATA_DIRS="/var/lib/flatpak/exports/share:/home/markus/.local/share/flatpak/exports/share" flatpak run org.libelektra.kdb --help                                                                                      Mittwoch 08:29:30
Sorry, I was not able to execute the man-page viewer: "/usr/bin/man".
Try to change /sw/elektra/kdb/#0/current/man with full path to man.

If you did not modify settings related to the man-page viewer,
please report the issue at https://issues.libelektra.org/
```

Reply:
`/usr/bin/man` seems to be available inside the package. I tried running `kdb help` during the build and it worked flawlessly, yet with the installed package, it does not. I am not sure as to why this happens, though. I researched it and wanted to see how others were doing it, but couldn't find anything specific. I may need help from somebody who is more fluent in flatpak.

Reply:
Yes, let us leave this for a follow-up.